plugin/wavesurfer.regions.js

```diff
@@ -269,19 +269,22 @@ WaveSurfer.Region = {
     bindInOut: function () {
         var my = this;
 
+        my.firedIn = false;
+        my.firedOut = false;
+
         var onPlay = function () {
-            my.firedIn = false;
-            my.firedOut = false;
```

Comment:
You don't have to re-submit the PR, just remove the function and push to the same branch, and this PR will be updated.